scripts/build.ps1

```diff
@@ -29,9 +29,9 @@ if (!$env:DOTNET_BUILD_VERSION) {
     $buildnumber = $commitTime.Days
     $revnumber = $commitTime.TotalSeconds
     
-    $VersionSuffix = "$buildnumber.$revnumber"
+    $VersionSuffix = "dev-$revnumber"
```

Comment:
let's discuss the versioning separately: https://github.com/dotnet/cli/issues/450